package.json

```
@@ -24,6 +24,7 @@
     "cookbook": "cd polyfill && npm install && npm run cookbook && cd ..",
     "test262": "cd polyfill && npm install && npm run test262",
     "lint": "eslint . --ext js,mjs",
+    "lint-fix": "eslint . --ext js,mjs --fix",
```

```suggestion
    "lint-fix": "npm run lint -- --fix”,
```
but I’d also suggest just running that locally instead of adding a script :-)